alert_rules/alert-rules.yml

```diff
@@ -5,5 +5,6 @@ groups:
     expr: dependency_up < 1
     for: 2m
     annotations:
+      app: "{{ $labels.prsn }}"
```

Comment:
@gutorc92, sure it's with quotes? `""`

Reply:
Without quotes prometheus cannot parse the rule file. It gives the error: 

> err="/etc/prometheus/alert-rules.yml: yaml: unmarshal errors:\n  line 8: cannot unmarshal !!map into string

Reply:
Ok